dvol_python/test_plugin.py

```diff
@@ -269,6 +269,40 @@ def cleanup():
         self.fail("Volume 'docker-volume-list-test' not found in Docker "
                 "output:\n\n" + docker_output)
 
+    def test_unique_volumes(self):
+        """
+        Two separate volumes do not share the same filesystem namespace.
+        """
+        alpha = "test-unique-volume-alpha"
+        beta = "test-unique-volume-beta"
```

Comment:
Thanks however I would like to keep the style the same as the rest of the tests.